app/controllers/videos_controller.rb

```diff
@@ -1,7 +1,8 @@
 class VideosController < ApplicationController
+  before_action :require_login
 
   def index
-    @videos = Video.all.limit 6
+    @categories = Category.all
     render :home
```

Comment:
I don't use unconventional template names in my apps because you want to easily map your files to controller actions. The less code you have to load in your head, the better.